README.md

```diff
@@ -1,2 +1,84 @@
-# content-providers
```

Comment:
Instead of having the Google Storage content provider take up the entire repository, you might want to try structuring it like this to support having other providers here.

```
google-storage/
aws-s3/
azure-storage/
dropbox/
README.md
```